protocol/x/rewards/keeper/keeper.go

```diff
@@ -126,26 +128,45 @@ func (k Keeper) GetRewardShare(
 //     by quote quantums of the fill.
```

Comment:
Let's modified comment according to [this](https://www.notion.so/dydx/RevShare-Generalization-6d89820112c94dfeaefc5a5ace052cf4?pvs=4#4d6688b828d34c0ea95cc6c3fcf1c31f)